### PR TITLE
Add more auto labeling

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Adds a couple more labels.

#### Describe the solution
Testing this to see if I can follow the template @casswedson created.

#### Describe alternatives you've considered
None

#### Testing
Has to be merged to test.

### CODE REVIEW

**Comment by Maleclypse** [comment]
@casswedson  Do you know if this 
```
"Player Faction Base / Camp":
  - src/basecamp**/*
```
Will work?

**Comment by casswedson** [comment]
This works
```yml
"Player Faction Base / Camp":
  - "src/basecamp.(cpp|h)"
# boy I am so bad at testing this >.<
```

**Comment by Maleclypse** [comment]
Reminder for once I find my computer to add CBMs tag for `[data/json/bionics.json]`

File: .github/labeler.yml [hunk 1]
```diff
@@ -18,6 +18,20 @@
 "Code: Test":
   - tests/**/*
 
+"NPC / Factions":
+  - data/json/npcs/**/*
+
+"CBMs":
```
Comment (casswedson) [inline .github/labeler.yml:24]:
```suggestion
"Bionics":
```
If you want it to use the existing `Bionics` label

Comment (Maleclypse) [reply .github/labeler.yml]: Thank you!

**Comment by Maleclypse** [comment]
To Fix `ERROR : src/mapgen.cpp:2598 [virtual void jmapgen_terrain::apply(const mapgendata &, const jmapgen_int &, const jmapgen_int &, const std::string &) const] In nested mapgen 5x5_gym_E in nested mapgen room_10x10_pool_W in mapgen house_detatched4_basement on house_detatched4_basement_west, setting terrain to t_carpet_yellow (from t_thconc_floor) at (21,4) when item towel existed.  Resolve this either by removing the terrain from this mapgen, adding suitable removal commands to the mapgen, or by adding a suitable flag to the innermost mapgen: either ERASE_ALL_BEFORE_PLACING_TERRAIN if you wish terrain to replace everything previously on the tile or ALLOW_TERRAIN_UNDER_OTHER_DATA if you wish the other items to be preserved133.202 s: overmap_terrain_coverage`

**Pull request merged**